addons/mail/data/mail_groups.xml

```diff
@@ -1,13 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-<data noupdate="1">
+
+    <data>
+
+        <record id="shortcode_manager" model="res.groups">
+            <field name="name">Mail canned response manager</field>
```

Comment:
Can we use this PR to decide once and for all if we want to use the term short code or canned response and use it everywhere?

Reply:
Which one do we want to use? I prefer shortcode because it's short…

Reply:
It's a functional decision with PO, the name we use should be the one we display to users.

Reply:
Hi @seb-odoo @phenix-factory 
That will be canned responses :)